### PR TITLE
Add load and store operations for SIMD types in runtime

### PR DESCRIPTION
**Pull request by chuan9**
BUG=XWALK-5108

### CODE REVIEW

**Comment by chuan9** [comment]
@huningxin ping~Please have a look.

File: src/runtime/runtime-array.cc [hunk 1]
```diff
@@ -958,7 +958,7 @@ RUNTIME_FUNCTION(Runtime_EstimateNumberOfElements) {
         ++holes;
       }
     }
-    int estimate = static_cast<int>((kNumberOfHoleCheckSamples - holes) /
+    int estimate = static_cast<int>((kNumberOfHoleCheckSamples - holes)/
```
Comment (huningxin) [inline src/runtime/runtime-array.cc]: We don't need this change.

**Comment by huningxin** [comment]
LGTM with one nit.

**Comment by chuan9** [comment]
@rakuco Done~Please have a review.

**Comment by rakuco** [comment]
merging. @mrunalk: please remember to roll `DEPS.xwalk`.

**Comment by mrunalk** [comment]
Done.

**Comment by rakuco** [comment]
@chuan9: was it intentional that there's no `simd_object` flag in this patch? Crosswalk uses it to enable SIMD.js support.

**Comment by chuan9** [comment]
@rakuco the flag is --harmony-simd, you can change it in xwalk_browser_main_parts.cc.

**Comment by rakuco** [comment]
OK, I've adjusted Crosswalk. However, you should have mentioned that beforehand, otherwise there was no way for us to know that was the case and we were going to have Crosswalk without SIMD.js support until somebody from QA noticed it.

**Pull request merged**